walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -12,12 +12,103 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Figure out what everything is 
+// Find events for attendies, to show conflicts
+// Sort by start time
+// Search for conflicts
+// TODO: Find times where attendies can meet without conflict (Time ranges for mettings without conflict)
+
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.List;
 
 public final class FindMeetingQuery {
+  // @param Event (Collection) - All the events existing on an organization calendar: name, time range, attendies...
+
+  // @param Meeting request - Meeting of a specific time with certain attendies (and duration)
+
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+
+      // Collection of required attendees
```

Comment:
Remove all of these extra comments at the top (look like they are from planning the algorithm?)